### PR TITLE
Cleared eslint issues

### PR DESCRIPTION
**Pull request by Rudraksha20**
@lilleyse cleared some eslint errors.

### CODE REVIEW

File: validator/specs/lib/validateTilesetSpec.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ describe('validateTileset', function() {
         expect(validateTileset(tileset)
             .then(function(message) {
                 expect(message).toBe('Tileset must declare a version in its asset property');
-            }), done).toResolve();
+           }), done).toResolve();
```
Comment (lilleyse) [inline validator/specs/lib/validateTilesetSpec.js]: Undo this change.

**Comment by lilleyse** [comment]
Looks good besides that one comment.

File: validator/specs/lib/validateTilesetSpec.js (hunk 1 above)
Comment (Rudraksha20) [inline validator/specs/lib/validateTilesetSpec.js]: Done.
